core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -500,7 +500,7 @@ private[ui] class JobPagedTable(
   override def tableId: String = jobTag + "-table"
 
   override def tableCssClass: String =
-    "table table-bordered table-condensed table-striped " +
+    "table table-bordered table-condensed table-striped sortable " +
```

Comment:
It doesn't sound like this just makes it wrap. I'm not sure what the implications are of this?